portfolio/src/main/java/com/google/sps/data/Comment.java

```diff
@@ -0,0 +1,84 @@
+package com.google.sps.data;
+
+import javax.servlet.http.HttpServletRequest;
+import com.google.appengine.api.datastore.Entity;
+
+/**
+ * A class for storing a Comment.
+ */
+public class Comment {
+    private String author;
+    private String text;
+    private long timestamp;
+
+    /**
+     * No argument constructor.
+     * Set fields with setter methods.
+     */
+    public Comment() {
+        this.timestamp = System.currentTimeMillis();
+    }
+
+    public Comment(String author, String text) {
+        this();
+        this.author = author;
+        this.text = text;
+    }
+
+
+    public long getTimestamp() {
+        return System.currentTimeMillis();
```

Comment:
get/setTimestamp doesn't follow the setter/getter expectations. 

.setTimestamp(10)
// .getTimestamp() should return 10, but will probably not.

Reply:
Thanks for the information. Changed to return instance variable instead.